packages/gestalt/src/Popover.js

```diff
@@ -64,6 +68,7 @@ type Props = {|
  * Popover is most appropriate for desktop screens and can contain a variety of elements, such as [Button](/button) and [Images](/image). Popover is also the container used to construct more complex elements like [Dropdown](/dropdown) and the board picker, pictured below, which allow people to choose the board to save a Pin to.
  */
 export default function Popover({
+  accessibilityLabel = 'Popover',
```

Comment:
A safe default label is always "Popover"

Reply:
not translated though… 😕

Reply:
Yeah, once we get the i10n going, we can definitely fix this